app/build.gradle

```diff
@@ -4,7 +4,7 @@ android {
     buildToolsVersion '23.0.2'
     defaultConfig {
         applicationId 'fr.neamar.kiss'
-        minSdkVersion 10
+        minSdkVersion 11
```

Comment:
@Cutta: I don't mean to step on your toes, but have you actually tested this (Android emulator)? We've had broken beta versions (and F-Droid) versions of KISS before :cry: